omnistrate_tests/test_replication_replicas.py

```diff
@@ -0,0 +1,217 @@
+import sys
+import signal
+from pathlib import Path
+from redis import Redis
+file = Path(__file__).resolve()
+parent, root = file.parent, file.parents[1]
+sys.path.append(str(root))
+
+# Additionally remove the current file's directory from sys.path
+from contextlib import suppress
+
+with suppress(ValueError):
+    sys.path.remove(str(parent))
+
+import logging
+import time
+import os
+from omnistrate_tests.classes.omnistrate_fleet_instance import OmnistrateFleetInstance
+from omnistrate_tests.classes.omnistrate_fleet_api import OmnistrateFleetAPI
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("omnistrate_user")
+parser.add_argument("omnistrate_password")
+parser.add_argument("cloud_provider", choices=["aws", "gcp"])
+parser.add_argument("region")
+
+parser.add_argument(
+    "--subscription-id", required=False, default=os.getenv("SUBSCRIPTION_ID")
+)
+parser.add_argument("--ref-name", required=False, default=os.getenv("REF_NAME"))
+parser.add_argument("--service-id", required=True)
+parser.add_argument("--environment-id", required=True)
+parser.add_argument("--resource-key", required=True)
+parser.add_argument("--replica-id", required=True)
+
+
+parser.add_argument("--instance-name", required=True)
+parser.add_argument("--instance-description", required=False, default="test-standalone")
+parser.add_argument("--instance-type", required=True)
+parser.add_argument("--storage-size", required=False, default="30")
+parser.add_argument("--tls", action="store_true")
+parser.add_argument("--rdb-config", required=False, default="medium")
+parser.add_argument("--aof-config", required=False, default="always")
+parser.add_argument("--replica-count", required=False, default="1")
```

Comment:
The api param replicacount has only 2 or 3 as options. Isn't it better to match the meaning, and force here to be 2 at minimum?